test/extended/machines/cluster.go

```diff
@@ -0,0 +1,49 @@
+package operators
+
+import (
+	"context"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	"github.com/stretchr/objx"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
+
+	"github.com/openshift/origin/test/extended/util/ibmcloud"
+)
+
+var _ = g.Describe("[sig-cluster-lifecycle][Feature:Machines][Early] Managed cluster should", func() {
+	defer g.GinkgoRecover()
+
+	g.It("have same number of Machines and Nodes", func() {
+		if e2e.TestContext.Provider == ibmcloud.ProviderName {
```

Comment:
This is just a special case of the "UPI/no-machineAPI" scenario that also applies in e.g. AWS UPI, so should be replaced by whatever check we use for that.